pkgs/development/python-modules/pygatt/default.nix

```diff
@@ -2,51 +2,56 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  fetchpatch2,
   mock,
-  nose,
   pexpect,
   pyserial,
+  enum-compat,
   pytestCheckHook,
   pythonOlder,
   setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "pygatt";
-  version = "4.0.5";
+  version = "4.0.5-unstable-2021-09-13";
   pyproject = true;
 
   disabled = pythonOlder "3.5";
 
   src = fetchFromGitHub {
     owner = "peplin";
     repo = "pygatt";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-DUZGsztZViVNZwmhXoRN5FOQ7BgUeI0SsYgCHlvsrv0=";
+    rev = "70c68684ca5a2c5cbd8c4f6f039503fe9b848d24";
+    hash = "sha256-u1kFw8JksWvMtAn1SqdcZeDs8kv5s0mdAj2+leXzMcc=";
   };
 
+  patches = [
+    # Migrate to pytest: https://github.com/peplin/pygatt/pull/351
+    (fetchpatch2 {
+      url = "https://github.com/peplin/pygatt/commit/467bd9620c73a17029c18b4beff8b1d38e46ec25.patch?full_index=1";
+      hash = "sha256-FR9ZoqdrQSbXjUDqHljlcfxMitbr7CDpVEJ3XuCAUoo=";
+    })
+  ];
+
+  pythonRemoveDeps = [ "coverage" ];
+
   postPatch = ''
-    # Not support for Python < 3.4
     substituteInPlace setup.py \
-      --replace-fail "'enum-compat'" "" \
-      --replace-fail "'coverage >= 3.7.1'," "" \
-      --replace-fail "'nose >= 1.3.7'" ""
-    substituteInPlace tests/bgapi/test_bgapi.py \
-       --replace-fail "assertEquals" "assertEqual"
+      --replace-fail '"coverage == 5.5", ' ""
```

Comment:
This should no longer be required, right?

Reply:
nope sadly